node/node.go

```diff
@@ -662,6 +662,7 @@ func validateShardBoundMessage(consensus *consensus.Consensus, nodeConfig *nodec
 }
 
 var (
+	errMsgHadNoHMYPayLoadAssumption      = errors.New("did not have sufficient size for hmy msg")
```

Comment:
optional:   maybe the var name can be without Msg part:  errHadNoHMYPayLoadAssumption

Reply:
errMsgHadNoHMYPayLoadAssumption removed, dead code.